survey-thank-you.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
```

Comment:
The text can overlap the green bits on certain widths. We should make sure there's enough margin/padding to prevent this.

<img width="901" alt="screen shot 2017-02-06 at 14 24 18" src="https://cloud.githubusercontent.com/assets/109850/22648664/00fd78e6-ec78-11e6-82f8-00aa3d18cb52.png">